src/libstd/ffi/os_str.rs

```diff
@@ -1174,6 +1175,15 @@ impl AsInner<Slice> for OsStr {
     }
 }
 
+#[stable(feature = "osstring_from_str", since = "1.45.0")]
+impl FromStr for OsString {
```

Comment:
What about OsStr?

Reply:
FromStr can't borrow.

Reply:
I see it now. Thanks for the clarification!